bootstrap/config/kfctl_iap.yaml

```diff
@@ -3,23 +3,46 @@
 # All TODO fields need to be changed following user's input before apply
 # TODO change repo on the fly: set it to local tmp dir containing kubeflow registry
 repo: /path/to/local/tmp/containing/kubeflow
-components:
-  - metacontroller
+packages:
+  - argo
+  - pipeline
+  - common
+  - examples
   - jupyter
+  - katib
+  - mpi-job
+  - pytorch-job
+  - seldon
+  - tf-serving
+  - openvino
+  - tensorboard
+  - tf-training
+  - metacontroller
+  - profiles
+  - application
+  - modeldb
+  - gcp
+components:
+  - pytorch-operator
   - ambassador
+  - openvino
+  - jupyter
+  - notebook-controller
+  - jupyter-web-app
```

Comment:
openvino is not a default component, shouldn't be part of default config.
 We can add it on the fly via go logic.

Reply:
removed.